scss/main.scss

```diff
@@ -90,10 +90,28 @@ $animation-length: 80s;
   background-color: #fff !important;
   box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.05);
   transition: background-color 200ms linear;
+
   .nav-link {
     color: $lavender !important;
     transition: color 200ms linear;
   }
+    
+  .logo {
+    background: linear-gradient(90deg, #00FFFF 0%, #AA96CA 46.35%, #FF6B98 100%);
```

Comment:
i feel like the orb colors are still harsh in contrast to pastel theme... i know it's official branding tho so not sure if can change anything

Reply:
lol yeah idont think theres anything we can do abt that :/